vignettes/Data_processing.Rmd

```diff
@@ -1496,6 +1477,43 @@ fullPlot
+numbering has been conserved from past <font face="CMTT10">tcpl</font> versions. $\mathit{Conc}$ = concentration; $\mathit{hitc}$ = hit call; $\mathit{|top|}$ =
+absolute value of the modeled curve top; $\mathit{coff}$ = cutoff; $\mathit{log_c(min)}$ = minimum log~10~ concentration tested;
+$\mathit{log_c(max)}$ = maximum log~10~ concentration tested; AC~50~ = $50 \%$ activity concentration; AC~95~ = $95 \%$ activity
+concentration.
+
+Fit category ($\mathit{fitc}$) is largely based upon the relative efficacy and, in the case of actives, the location of the AC~50~ and
+concentration at $95 \%$ activity (an estimate of maximum activity concentration, AC~95~) compared to the
+tested concentration range. Figure 5 illustrates the $\mathit{fitc}$ hierarchical tree used to determine the $\mathit{fitc}$. All
+concentration response curves are first split into active, inactive, or cannot determine. “Cannot
+determine” is indicative of exceptions that cannot be curve-fit, e.g. a concentration series with fewer than
+4 concentrations or an apparent negative response within a positively-analyzed, unidirectional endpoint.
+Active designations are determined for $\mathit{fitc}$ based on whether the $\mathit{hitc}$ surpasses the 0.90 threshold. For
+those series that are designated inactive with a $\mathit{hitc}$ less than 0.90, $\mathit{fitc}$ can be used to indicate to what
+extent the curve represents borderline inactivity via comparison of top modeled efficacy to the cutoff (i.e.,
+the absolute value of the modeled top is less than 0.8 times the cutoff).
+
+For active curves, efficacy, as represented by the modeled top, is compared to 1.2 times the cutoff
+(less than or equal to, or greater than), thereby differentiating curves that may represent borderline
+activity from moderate activity. Active curves also have potency metrics estimated, e.g., AC~50~ and AC~95~
+values, that can be compared to the range of concentrations screened to indicate curves for which
+potency estimates are more quantitatively informative. Curves for which the AC~50~ is less than or equal to
+the minimum concentration tested ($\mathit{fitc}$ = 36, 40) may indicate AC~50~ values that are less quantitatively
+informative than AC~50~ values within the concentration range screened. When the AC~50~ is greater than
+the minimum concentration tested but the AC~95~ is greater than or equal to the maximum concentration
+tested ($\mathit{fitc}$ = 38, 42), it is possible the maximum activity was not fully observed in the concentration range
+screened. $\mathit{Fitc}$ for curves where the AC~50~ and AC~95~ are both within the concentration range screened ($\mathit{fitc}$
+= 37, 41) represent the most quantitatively informative AC~50~ values. In some previous applications, curves
+with the modeled top less than or equal to 1.2 times the cutoff and an AC~50~ less than the concentration
```

Comment:
Remove last sentence then add this new paragraph to help with interpretation: "Fitc 36 describes a curve that is of low efficacy and with a low AC50, below the concentration range screened. These are more likely to be noise or less reproducible fits. Fitc 41 and 42 are the ideal fitc for reproducible curves, as demonstrated by these two fitc comprising the majority of positive (hitc > 0.9) curves in invitrodb v4.1. Fitc 40 indicates a curve with at least moderate efficacy, but an AC50 below the concentration range screened. These chemicals may be positive or reference chemicals screened in the incorrect concentration window to observe their minimum activity. These curves may also represent high-confidence positives for which we have limited understanding of the slope of the concentration-response curve, and as such, the AC50 may be associated with more uncertainty."